experiment/run_experiment.py

```diff
@@ -96,10 +96,6 @@ def _validate_config_parameters(
     for param in missing_params:
         logs.error('Config does not contain required parameter "%s".', param)
 
-    # Notify if any optional parameters are missing in config.
-    for param in optional_params:
-        logs.info('Config does not contain optional parameter "%s".', param)
```

Comment:
I used this to double-check that all my parameters are passed correctly.
It's ok to remove it.